packages/webapp/pages/onboarding.tsx

```diff
@@ -84,12 +101,52 @@ interface HeaderProps {
   showOnboardingPage: boolean;
   isOnboardingV3: boolean;
   setAuth: Dispatch<SetStateAction<AuthProps>>;
+  onClickNext: () => void;
+  isFiltering: boolean;
 }
 const Header = ({
   showOnboardingPage,
   isOnboardingV3,
+  isFiltering,
   setAuth,
+  onClickNext,
 }: HeaderProps) => {
+  const onboardingV4 = useFeature(feature.onboardingV4);
+  const isMobile = !useMedia([tablet.replace('@media ', '')], [true], false);
+  const isTablet = !useMedia([laptop.replace('@media ', '')], [true], false);
+
+  const getImage = () => {
+    if (isMobile) {
+      return cloudinary.feed.bg.mobile;
+    }
+
+    return isTablet ? cloudinary.feed.bg.tablet : cloudinary.feed.bg.laptop;
+  };
+
+  if (isFiltering && onboardingV4 === OnboardingV4.V4) {
+    return (
+      <header className="flex sticky top-0 z-3 justify-center mb-10 w-full backdrop-blur-sm">
+        <img
+          className="absolute top-0 right-0 left-0 w-full pointer-events-none"
+          src={getImage()}
+          alt="Gradient background"
+        />
+        <div className="flex justify-between items-center py-10 w-full max-w-4xl tablet:!px-6 !px-4">
+          <Logo
+            logoClassName={classNames(
+              onboardingV4 === OnboardingV4.V4 && 'h-6',
+            )}
+            position={LogoPosition.Relative}
+          />
+          <CreateFeedButton
+            requiredTagsThreshold={requiredTagsThreshold}
```

Comment:
Why are we passing this down, is it not always 5?

Reply:
I need it on couple of places so exporting from here and passing down.